src/plugins/cordova-plugin-device-orientation/sim-host.js

```diff
@@ -32,7 +32,7 @@ module.exports = function (messages) {
         });
         compassWidget.initialize(compass.heading);
 
-        inputHeading.addEventListener('input', function () {
+        inputHeading.addEventListener('change', function () {
```

Comment:
Right, I'll remove these changes from here and add them to my other active branch where I'm already doing telemetry changes

Thanks!